src/GeneralStateTestsFiller/stSubroutine/dummyFiller.json

```diff
@@ -0,0 +1,106 @@
+{
+    "dummy" : {
```

Comment:
wait, I thought we are removing this folder for good from develop. it can stay removed.
retesteth will just show a warning.  the sanitizer Makefile script can filter it out. (because of the git issue)

Reply:
I figured I'll leave this to avoid problems until you remove stSubroutine from retesteth, and then delete it. Is it already removed?